### DISCUSSION TITLE
install sources - filenames relevant ?

### DISCUSSION THREAD

**Discussion by MagnaFrisia90** [HCL-TECH-SOFTWARE/connections-automation#256] [Q&A] [answered]
Hi,
when offering the files with a ruby/python-http server: Do the sources need specific strings ? 
Filename in Flexnet for Websphere sources :  CIK2IML.zip
Filename in documentation example: WAS_ND_V8.5.5_1_OF_3.zip
(not sure if CIK2IML is actually WASND 1outof3 but this is only for example)

Do I need to rename all the flexnet sources to a specific naming ?

And regarding V8:  It seems like Component Pack ist not anymore available as .zip file. I assume the Ansible magic will just pull helm charts from https://hclcr.io/chartrepo/cnx which contain all required stuff ?

Regards,
Jan

**Comment by sabrina-yee** [comment, marked as answer]
The filenames can be overridden like this: 
https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/84883bf2986b05643857d6255cc2cea5fc92d97a/environments/examples/cnx7/flexnet_db2/group_vars/all.yml#L54-L62

As for the the Component Pack zip file, that's right it is no longer a zip file and the latest can be found on the HCL Harbor repository on https://hclcr.io/chartrepo/cnx which is used by this Ansible repo to pull the charts from there via Helm.

**Reply by jvaldman** [reply]
Can somebody update README.md then, please?
